src/snyk/common/vscode/languageConsts.ts

```diff
@@ -0,0 +1,9 @@
+export const TYPESCRIPT_FILE_REGEX = new RegExp('\\.tsx?$');
```

Comment:
Please don't duplicate the code. Use 'common' folder to move the existing code to, if shared between different products.

Reply:
I'm not sure I understand, it's already in common.

Reply:
see src/snyk/snykOss/constants/language.ts.

Reply:
Damn, looks like I created common files but forgot to delete the old ones.